framework/core/locale/core.yml

```
@@ -350,7 +350,7 @@ core:
     forgot_password:
       dismiss_button: => core.ref.okay
       email_placeholder: => core.ref.email
-      email_sent_message: We've sent you an email containing a link to reset your password. Check your spam folder if you don't receive it within the next minute or two.
+      email_sent_message: If this user exists, we've sent an email containing a link to reset your password. Check your spam folder if you don't receive it within the next minute or two.
```

The wording feels a bit clunky. I think it's partially because we switch from "if this user exists" to "your password" in one sentence. How does this sound?

```suggestion
      email_sent_message: If this email is registered, we'll send an email containing a link to reset their password. Check your spam folder if you don't receive it within the next minute or two.
```